ckanext/canada/helpers.py

```diff
@@ -505,19 +508,19 @@ def json_loads(value: str) -> Dict[str, Any]:
 
 
 def get_datapreview(res_id: str) -> str:
-    dsq_results = ckan.logic.get_action('datastore_search')(
-        {}, {'resource_id': res_id, 'limit': 100})
+    dsq_results = t.get_action('datastore_search')(
```

Comment:
we *really* shouldn't be using `t` as a global variable for ckan.toolkit. One-letter variables are almost always short-lived loop variables and the like.

There is precedence for using `tk` for the toolkit module, so we could change it to that.

Reply:
I personally hate the use of `t` alias for toolkit so thank god hahaha. I also don't like `tk`, but that is just my personaly preference because I am weird. (* continues to use underscored named variables in python)

I will just do `toolkit`

@wardi in the topic of `toolkit` do we even need this anymore: https://github.com/ckan/ckantoolkit

Reply:
Okay, `t` fixed up here and just used imports from `ckan.plugins.toolkit`

So just the big 'ol question about the `ckantoolkit` repo

Reply:
All the compatibility fixes in ckantoolkit were for long forgotten versions of ckan. We can remove it.